docs/reference/src/code/language/program-types/contracts/wallet/Forc.toml

```diff
@@ -5,7 +5,7 @@ license = "Apache-2.0"
 name = "wallet"
 
 [constants]
```

Comment:
we can get rid of this as well

Reply:
Sorry this one isn't as clear to me. Are we removing line 5 and 7? or just one of them?

Reply:
just the unused `[constants]`